pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -76,6 +77,24 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 
 var _ reconcile.Reconciler = &ReconcileHyperConverged{}
 
+func (r *ReconcileHyperConverged) updatePhase(instance *hcov1alpha1.HyperConverged, phase hcov1alpha1.HyperConvergedPhase, request reconcile.Request, failedOperation string) error {
+	err := r.client.Get(context.TODO(), request.NamespacedName, instance)
+	if err != nil {
+		return err
+	}
+	instance.Status.Phase = phase
+	err = r.client.Status().Update(context.TODO(), instance)
```

Comment:
Does this retrigger reconciliation? Maybe you need to add a predicate to reconciliation, to ignore status updates https://github.com/kubevirt/cluster-network-addons-operator/blob/master/pkg/controller/networkaddonsconfig/networkaddonsconfig_controller.go#L107

Reply:
My understanding is that using the [status subresource on the CRD](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/deploy/converged/crds/hco.crd.yaml#L23-L24) makes it so we can independently update the status w/o triggering a reconcile.

Reply:
I see, right!